modules/core/src/main/java/org/apache/ignite/internal/IgnitionEx.java

```diff
@@ -2189,6 +2190,11 @@ private IgniteConfiguration initializeConfiguration(IgniteConfiguration cfg)
 
             myCfg.setNodeId(nodeId);
 
+            String predefineConsistentId = IgniteSystemProperties.getString(IGNITE_CONSISTENT_ID);
+
+            if (!F.isEmpty(predefineConsistentId))
```

Comment:
predefineConsistentId should not take precedence over the one specified in configuration. Please only use predefineConsistentId if no consistentId is supplied in cfg.

Reply:
I am sure that I mean the overwriting property.
I think, it is general approach in Ignite (see IGNITE_OVERRIDE_MCAST_GRP, IGNITE_REFLECTION_CACHE_SIZE, IGNITE_LOCAL_HOST, IGNITE_DEP_MODE_OVERRIDE).

Reply:
Please rename it to IGNITE_OVERRIDE_CONSISTENT_ID so that behavior is apparent

Reply:
Done.

Reply:
Renamed